examples/demo/fxos-text-input.js

```diff
@@ -1 +0,0 @@
-../../fxos-text-input.js
```

Comment:
These file used to be symlinked so that you could develop against the demo app. IIUC we now have to `npm install` inside the demo app to get the new source. Could we symlink `gaia-text-input/` into `examples/demo/node_modules/`?